e-commerce/shop-backend/dbos-config.yaml

```diff
@@ -12,8 +12,8 @@ database:
 runtimeConfig:
   port: 8082
 application:
-  payment_host: 'http://localhost:8086'
-  local_host: 'http://localhost:8082'
+  # payment_host: 'http://localhost:8086'
+  # local_host: 'http://localhost:8082'
   # cloud example
-  # payment_host: 'https://dbos_domain/dbos-testuser/application/payment-backend'
-  # local_host: 'https://dbos_domain/dbos-testuser/application/shop-backend'
+  payment_host: 'https://dbos_domain/dbos-testuser/application/payment-backend'
```

Comment:
Should `dbos_domain` be an environment variable?

Reply:
For staging definetly yes.
Can our SDK replace within strings. An early test did not seem to work.
Otherwise we need SHOP_PAYMENT_HOST and LOCAL_PAYMENT_HOST.